src/app/datagrid/selection-row-mode/selection-row-mode.html

```diff
@@ -0,0 +1,122 @@
+    </li>
+</ul>
+
+<p>
+    In the following examples, we simply display the names of the selected users
+</p>
+
+<div class="card card-block">
+    <p class="card-text username-list">
+        Selected user:
+        <em *ngIf="!singleSelected">No user selected.</em>
+        <span class="username" *ngIf="singleSelected">{{singleSelected.name}}</span>
+    </p>
+</div>
+
+<h3>Single selection</h3>
+
+<clr-datagrid [(clrDgSingleSelected)]="singleSelected" clDgRowSelection="true">
+
+    <clr-dg-column>User ID</clr-dg-column>
+    <clr-dg-column>Name</clr-dg-column>
+    <clr-dg-column>Creation date</clr-dg-column>
+    <clr-dg-column>Favorite color</clr-dg-column>
+
+    <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+        <clr-dg-cell>{{user.id}}</clr-dg-cell>
+        <clr-dg-cell>{{user.name}}</clr-dg-cell>
+        <clr-dg-cell>{{user.creation | date}}</clr-dg-cell>
+        <clr-dg-cell>
+            <span class="color-square" [style.backgroundColor]="user.color"></span>
+        </clr-dg-cell>
+    </clr-dg-row>
+
+    <clr-dg-footer>{{users.length}} users</clr-dg-footer>
+</clr-datagrid>
+
+<clr-example [clrCode]="singleSelectionExample" clrLanguage="html"></clr-example>
+
+<h3>Multi selection</h3>
+
+<div class="card card-block">
+    <p class="card-text username-list">
+        Selected users:
+        <em *ngIf="multiSelected.length == 0">No user selected.</em>
+        <span class="username" *ngFor="let user of multiSelected">{{user.name}}</span>
+    </p>
+</div>
+
+<clr-datagrid [(clrDgSelected)]="multiSelected" clDgRowSelection="true">
+    <clr-dg-column>User ID</clr-dg-column>
+    <clr-dg-column>Name</clr-dg-column>
+    <clr-dg-column>Creation date</clr-dg-column>
+    <clr-dg-column>Favorite color</clr-dg-column>
+
+    <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+        <clr-dg-action-overflow>
+            <button class="action-item">Action</button>
+        </clr-dg-action-overflow>
+
+        <clr-dg-cell>{{user.id}}</clr-dg-cell>
+        <clr-dg-cell>{{user.name}}</clr-dg-cell>
+        <clr-dg-cell>{{user.creation | date}}</clr-dg-cell>
+        <clr-dg-cell>
+            <span class="color-square" [style.backgroundColor]="user.color"></span>
+        </clr-dg-cell>
+    </clr-dg-row>
+
+    <clr-dg-footer>{{users.length}} users</clr-dg-footer>
+</clr-datagrid>
+
+<clr-example [clrCode]="multiSelectionExample" clrLanguage="html"></clr-example>
+
+<h3>Multi selection with hidden selection column</h3>
+
+<div class="card card-block">
+    <p class="card-text username-list">
+        Selected users:
+        <em *ngIf="hiddenMultiSelected.length == 0">No user selected.</em>
+        <span class="username" *ngFor="let user of hiddenMultiSelected">{{user.name}}</span>
+    </p>
+</div>
+
+<clr-datagrid [(clrDgSelected)]="hiddenMultiSelected" clDgHideSelectionColumn="true">
```

Comment:
I disagree, because:
* Select all functionality (Checkbox in the first column's header) won't be supported, but also might need such a feature
* You might want to be explicit when showing the selection, so sometimes you might want to have row selection + checkboxes as a first column. This is widely use in our case when we show datagrids in dialogs.